Islandora/islandora_ocr
# 7.x islandora 2012

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-2012

# What does this Pull Request do?
Makes it so that the "generate_ocr" flag is used for OCR generation, and a new "generate_hocr" flag is used for HOCR generation.

# What's new?
* OCR/HOCR generation forced behaviour has been split between a 'generate_ocr' flag and a 'generate_hocr' flag
* An update hook has been added to apply current 'generate_ocr' flags on items to the 'generate_hocr' flag; that way, current behaviour is not interrupted on existing objects.

# How should this be tested?
The update hook should be tested to make sure that existing items have the 'generate_ocr' flag to them.
Other than that, most things to be tested UI/Drush-wise are elsewhere.

# Additional Notes:
* Despite being a fairly small update, this does fundamentally change the way we force OCR/HOCR-generating behaviour. Code that sets the 'generate_ocr' flag expecting to have that setting applied to HOCR now needs to have it split out. PRs are being made to paged content, book, and book batch, which will be linked to this.

# Interested parties
@jordandukart and @ajstanley maintain this so pinging e'm
